.github/workflows/pull-requests.yml

```diff
@@ -140,11 +140,20 @@ jobs:
           PERFORMANCE_GRAPHS_PLOTLY_API_KEY: ${{secrets.PERFORMANCE_GRAPHS_PLOTLY_API_KEY}}
         run: |
           nix-shell --arg includeServerBuildSupport false --arg includeRunLocallySupport false --run "run-puppeteer-test"
-      - name: Create comment
+      - name: Find Comment
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Compare with last deployed Master
```

Comment:
Rather than this, can we add a more explicit prefix to the actual comment as part of the workflow step below (e.g. `Performance test results:`), otherwise this will break if we change the output of the puppeteer script.

Reply:
fixed!